cds-feature-attachments/src/main/java/com/sap/cds/feature/attachments/service/handler/DefaultAttachmentsServiceHandler.java

```diff
@@ -81,7 +78,7 @@ public void restoreAttachment(AttachmentRestoreEventContext context) {
 	@On(event = AttachmentService.EVENT_READ_ATTACHMENT)
 	@HandlerOrder(HandlerConstants.DEFAULT_ON)
 	public void readAttachment(AttachmentReadEventContext context) {
-		logger.info(read_marker, "Default Attachment Service handler called for reading attachment with document id: {}",
+		logger.debug(attachmentServiceMarker, "Default Attachment Service handler called for reading attachment with document id {}",
```

Comment:
Log level not consistent, the other logs are on info, this is on debug level. Maybe all should be on debug ?